src/Gui/Windows/CombinedCodeViewInteractor.cs

```diff
@@ -299,12 +304,39 @@ public bool ChooseTextEncoding()
             return true;
         }
 
-        private void MixedCodeDataView_MouseDown(object sender, EventArgs e)
+        private void MixedCodeDataView_MouseUp(object sender, MouseEventArgs e)
+        {
```

Comment:
> In fact, as I write this, it makes total sense to integrate TypeMarker and DeclarationTextBox; they're almost doing the same thing, except that TypeMarker is using a dialect of "Hungarian" for speed and convenience.

John, what do you mean? Now I'm going to implement global variable declaration using C Parser. Is it enough? Or you suggest to use Hungarian instead (or in addition to).